### ISSUE TITLE
Login Screen on IOS does not scroll down to the Login button

### ISSUE THREAD

**Issue by gedw99** [jech/galene#111]
On IOS, when you try to scroll to the bottom to press the Login button, you can never get there.
The purple app bar does scroll up.

You cant see the problem in the screen shot because its a scrolling issue:

<img width="320" alt="galene-ios-login-bug" src="https://user-images.githubusercontent.com/53147028/134284089-e4809b04-4ffe-40ec-badf-f3f680e92f9e.png">

On Desktop when in chrome inspector it does work though:

![galene-ios-login-bug-in chrome-inspector](https://user-images.githubusercontent.com/53147028/134284617-5154be1f-04ea-488b-af8a-1b0a65e1f64e.png)

**Comment by jech** [comment]
I've seen the same issue with a small Android device.  The login screen is too large for small devices, and disables scrolling.

Three solutions: move everything upwards, make the login screen more reactive, or allow scrolling.  @takdj ?

**Comment by takdj** [comment]
I see the problem, it comes from here: https://github.com/jech/galene/blob/master/static/galene.css#L591
```
margin: 5em auto;
```

We can change to `margin: auto;` and the box will be center vertically, but it looks less good on Ipad in my opinion.
Scrool is also missing on login box here: https://github.com/jech/galene/blob/master/static/galene.css#L1264

```
.login-container {
   ...
    overflow: scroll;
}
```
@jech can you try if it's ok for you ?

**Comment by jech** [comment]
@takdj, yes, it works for me.

@gedw99 The proposed change is live on https://galene.org:8443.  Ok?

**Comment by jech** [comment]
Fixed in 5b1bc81.

**Comment by gedw99** [comment]
Thanks it works now
